src/app/shared/services/notifications/notifications.service.ts

```diff
@@ -28,16 +28,16 @@ export class NotificationsService {
    * This method read notifications by notifications type
    * @param type: string
    */
-  readUsersNotificationsByType(notificationsGrouped: NotificationGrouped): Observable<Object> {
-    return this.http.put<Object>(`/api/v1/Notification/ReadUsersNotificationsByType/${notificationsGrouped.type}`, notificationsGrouped);
+  readUsersNotificationsByType(notificationsGrouped: NotificationGrouped): Observable<any> {
+    return this.http.put<any>(`/api/v1/Notification/ReadUsersNotificationsByType/${notificationsGrouped.type}`, notificationsGrouped);
   }
 
   /**
    * This method read notification by notification id
    * @param id: string
    */
-  readUsersNotificationById(notification: Notification): Observable<Object> {
-    return this.http.put<Object>(`/api/v1/Notification/Read/${notification.id}`, notification);
+  readUsersNotificationById(notification: Notification): Observable<any> {
```

Comment:
I created an issue to fix this code smell further https://github.com/ita-social-projects/OoS-Frontend/issues/1676